clientSide/src/parse-functions/bookingComponentFunctions.js

```diff
@@ -44,7 +55,28 @@ export const getCarGroupPointer = async (carGroupName) => {
     console.error(ErrorLabels.carGroup, error);
   }
 };
-
+export const getCarPointer = async (carId) => {
```

Comment:
You have get CarPointer here and you call it inside updateFunctions.js. Maybe include it in this file instead?
Same for the next function. getCustomerPointer(). As I can see it is not called somewhere, but if it's called it is not in this file.